targets/TARGET_STM/gpio_irq_api.c

```diff
@@ -501,6 +505,10 @@ void gpio_irq_enable(gpio_irq_t *obj)
 
     LL_EXTI_EnableIT_0_31(1 << pin_index);
 
+#if defined(DUAL_CORE) && defined(CORE_CM4)
+    LL_C2_EXTI_EnableIT_0_31(1 << pin_index);
+#endif
+
```

Comment:
In case of dualcore CM4, I think we should enabe only CM4 interrupt
```suggestion
#if defined(DUAL_CORE) && defined(CORE_CM4)
    LL_C2_EXTI_EnableIT_0_31(1 << pin_index);
#else
    LL_EXTI_EnableIT_0_31(1 << pin_index);
#endif

```